--- src/router.tsx
+++ src/router.tsx
@@ -1,7 +1,8 @@
-import DefinitionsPage from "./pages/DefinitionsPage"
-import { createRootRoute, createRoute, createRouter } from '@tanstack/react-router'
-
-import { Link } from '@tanstack/react-router'
+import { createRootRoute, createRoute, createRouter, Link } from '@tanstack/react-router'
+
+import App from './App'
+import { FiguresNetwork } from './pages/FiguresNetwork'
+import DefinitionsPage from './pages/DefinitionsPage'

 const NotFound = () => {
   return (
@@ -21,9 +22,6 @@
     </div>
   )
 }
-
-import App from './App'
-import { FiguresNetwork } from './pages/FiguresNetwork'

 const rootRoute = createRootRoute({ notFoundComponent: NotFound })
